english/magic-device/magic-device.md

```diff
@@ -0,0 +1 @@
+It is then when you realise that such thing has to contain some magic, so you don't think it twice and grab it. Then, all at the sudden 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.